### ISSUE TITLE
Which one of all the Dotnet i need to install?

### ISSUE THREAD

**Issue by ZeroCool22** [Stability-AI/StableSwarmUI#15]
![Screenshot_2](https://github.com/Stability-AI/StableSwarmUI/assets/13344308/ecd99954-1882-4d45-99cb-e067395bbb12)

**Comment by ZeroCool22** [comment]
This one?

![Screenshot_3](https://github.com/Stability-AI/StableSwarmUI/assets/13344308/e90dcd13-6243-4dfe-b9fd-0a6235ca9f2f)

**Comment by mcmonkey4eva** [comment]
Yes, .NET SDK 7, that looks correct (sorry this is meant to install itself, but something bugs with the self-install so it's temporarily manual)
